client/src/components/payment/fee-summary.vue

```diff
@@ -161,25 +163,23 @@ header {
   }
 }
 
-.container.fee-total {
-  font-weight: bold;
-}
-
-.fee-list__item-name {
+.container.payment-total {
   font-weight: bold;
+  color: $gray9;
 }
 
 .container.fee-total,
 .container.fee-list__item {
   justify-content: space-between;
 }
 
-.container.fee-list__item {
-  border-bottom: 1px dotted grey;
+.header {
+  font-size: 1.125rem;
+  color: $gray9;
 }
 
-.fee-total,
+.header,
 .tax-total {
-  border-bottom: 1px dotted grey;
+  border-bottom: 1px solid #80808029;
```

Comment:
This doesn't look like a legitimate color (4 bytes)?

Also if possible, use a CSS variable instead of hex value.